org.gnunet.Messenger.json

```diff
@@ -0,0 +1,164 @@
+{
+  "app-id": "org.gnunet.Messenger",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "41",
+  "sdk": "org.gnome.Sdk",
+  "command": "messenger-gtk",
+  "finish-args": [
+    "--share=ipc",
+	"--socket=wayland",
+	"--share=network",
+	"--socket=pulseaudio",
+	"--device=dri",
+
+	/* Home directory access to share files */
+	"--filesystem=home",
+	
+	/* Webcam access to scan QR codes */
+	"--device=all"
+  ],
+  "modules": [
+    {
+	  "name": "recutils",
+	  "buildsystem": "autotools",
+	  "sources": [
+	    {
+		  "type": "archive",
+		  "url": "https://ftp.gnu.org/gnu/recutils/recutils-1.8.tar.gz",
+		  "sha512": "607284f3e0f067c4fbbd239ff93fdadcf0864d45cb7d27c955a79595778cd76531043f6896f00345fc74853c5fd63bd2379deb4f7d26d49cf515ae46a2dd937c"
+		}
+	  ]
+	},
+	{
+	  "name": "jansson",
+	  "buildsystem": "autotools",
+	  "sources": [
+	    {
+		  "type": "archive",
+		  "url": "https://digip.org/jansson/releases/jansson-2.13.tar.gz",
+		  "sha512": "79e26745e257118d4860cbfd57e250834c0766523d7f5c3d9e34438805f9c3064a5890ba3108292e5f2cb24d77a77049ffddd17332c65c293b70fb91869b67e6"
+		}
+	  ]
+	},
+	{
+	  "name": "libsodium",
+	  "buildsystem": "autotools",
+	  "sources": [
+	    {
+		  "type": "archive",
+		  "url": "https://download.libsodium.org/libsodium/releases/libsodium-1.0.18-stable.tar.gz",
+		  "sha512": "f57c41fddbb6b1598027f04f592d3860d403e1d0dfee146b550ab587eefb984108741e9af4b2687deba2fdc92911696ecf4005fb7c7d09cc78860394120db816"
+		}
+	  ]
+	},
+    {
+	  "name": "gnunet",
+	  "buildsystem": "autotools",
+	  "sources": [
+	    {
+		  "type": "archive",
+		  "url": "http://ftpmirror.gnu.org/gnunet/gnunet-0.16.3.tar.gz",
```

Comment:
https?

Reply:
Yes, I change that. I think I copied the link without https and later on I didn't think it would matter as much since we already confirm the archive via checksum.